src/components/Questions/QuestionModal.vue

```diff
@@ -15,6 +15,18 @@
     <div
       class="flex flex-col grow bg-white w-full justify-between overflow-hidden"
     >
+    <div class="grid grid-rows-1 grid-flow-col gap-2 ">
+          <div class="row-span-1 text-left h-10 mx-4 py-3 text-xl text-base sm:text-xs md:text-base lg:text-lg xl:text-xl 2xl:text-2xl" style = "@media (max-width: 870px){font-size: 0.75rem; line-height: 1rem;}">
```

Comment:
why are there tailwind classes along with an inline style attribute? 
`style = "@media (max-width: 870px){font-size: 0.75rem; line-height: 1rem;}">`
You can rewrite this thing using tailwind itself. You can extend the tailwind config and put a custom breakpoint. Take a look at the `tailwind.config.js` file in this project and see how we've added custom breakpoints. Similarly you can add the 870px waala breakpoint and then set the font size and line height